bridge-sdk/connectors/omni-connector/src/omni_connector.rs

```diff
@@ -179,6 +177,43 @@ impl OmniConnector {
         Ok(tx.tx_hash())
     }
 
+    pub async fn near_deploy_token(&self, chain_kind: ChainKind, vaa: &str) -> Result<CryptoHash> {
+        let near_endpoint = self.near_endpoint()?;
+        let token_locker_id = self.token_locker_id()?;
+
+        let args = omni_types::prover_args::WormholeVerifyProofArgs {
+            proof_kind: omni_types::prover_result::ProofKind::LogMetadata,
+            vaa: vaa.to_owned(),
+        };
+
+        let args = DeployTokenArgs {
+            chain_kind,
+            prover_args: borsh::to_vec(&args).unwrap(),
+        };
+
+        let mut serialized_args = Vec::new();
+        args.serialize(&mut serialized_args)
+            .map_err(|_| BridgeSdkError::UnknownError)?;
+
+        let tx_id = near_rpc_client::change(
+            near_endpoint,
+            self.near_signer()?,
+            token_locker_id.to_string(),
+            "deploy_token".to_string(),
+            serialized_args,
+            300_000_000_000_000,
+            4_000_000_000_000_000_000_000_000,
```

Comment:
Do we really need that much attached deposit and has here?

Here's an example of `deploy_token` tx:
<img width="578" alt="image" src="https://github.com/user-attachments/assets/5e2b2f66-3f0c-4b32-a164-23549d9ddeb8">

We only used 60 TGas and 4 NEAR looks like a big overkill. I believe that 1-2 NEAR should be enough to cover a storage cost

Reply:
I believe new token account requires 3 NEAR deposit

Reply:
Then why we attach 4 here and why 300 TGas?

Reply:
@karim-en how much would you recommend to attach for deploying the token? I think it's slightly more than 3 NEAR for the token account

Reply:
It looks like a hard const to me, so I thought that we can always attach exactly 3 NEAR. Am I wrong? If it's a bit more than 3, then you're correct and there's no point in attaching less and calculating these decimals. We can leave as it is 4 NEARs and everything else will be refunded in any case

<img width="496" alt="image" src="https://github.com/user-attachments/assets/f19c3a22-9c12-4c53-8608-17efe9486fe7">

Reply:
For now let's hardcode it as 4.
I'm going to improve it by reducing this amount and add a refund.